docs/readthedocs/source/doc/Nano/QuickStart/tensorflow_inference.md

```diff
@@ -1,3 +1,91 @@
 # BigDL-Nano TensorFlow Inference Overview
```

Comment:
`bigdl.nano.tensorflow.keras.Model` -> `bigdl.nano.tf.keras.Model`

And also mention `Sequential`?